src/background/browser-adapter.ts

```diff
@@ -24,7 +24,7 @@ export interface BrowserAdapter extends ClientBrowserAdapter {
     createInactiveTab(url: string, callback: (tab: chrome.tabs.Tab) => void): void;
     closeTab(tabId: number): void;
     switchToTab(tabId: number): void;
-    getTab(tabId: number, callback: (tab: chrome.tabs.Tab) => void): void;
+    getTab(tabId: number, onResolve: (tab: chrome.tabs.Tab) => void, onReject?: () => void): void;
```

Comment:
There are 4 other callsites for `BrowserAdapter.getTab` (across `details-view-initializer.ts`, `assessment-store.ts`, `tab-controller.ts`, and `target-tab-finder.ts`) that should probably be updated to account for this signature change.